arcgis-ios-sdk-samples/Route & Directions/Find closest facility to an incident (interactive)/README.md

```diff
@@ -15,14 +15,14 @@ Tap near any of the hospitals and a route will be displayed from that tapped loc
 ## How it works
 
 1.  Create a new `AGSClosestFacilityTask` using a URL from an online network analysis service.
-1.  Get `AGSClosestFacilityParameters` from the task.
-1.  Add facilities to the parameters.
-1.  Add an incident (as an `AGSPoint`) to the parameters.
-1.  Get `AGSClosestFacilityResult` by solving the task with the parameters.
-1.  Get the indexed list of closet facilities to the incident.
-1.  Get the index of the closest facility.
-1.  Get closest facility route from the facility result.
-1.  Display the route on the `AGSMapView` as an `AGSGraphic` on an `AGSGraphicsOverlay`.
+2.  Get `AGSClosestFacilityParameters` from the task.
+3.  Add facilities to the parameters.
+4.  Add an incident (as an `AGSPoint`) to the parameters.
+5.  Get `AGSClosestFacilityResult` by solving the task with the parameters.
+6.  Get the indexed list of closet facilities to the incident.
+7.  Get the index of the closest facility.
+8.  Get closest facility route from the facility result.
+9.  Display the route on the `AGSMapView` as an `AGSGraphic` on an `AGSGraphicsOverlay`.
```

Comment:
Are these changes fixing a bug or are they just style updates?

Reply:
For this one I consider it sort of a "bugfix", as the site generator/renderer that relies on the source from README might run into issues if the numbered list is written in `GitHub markdown flavor` rather than the standard format, as Zack from dotnet mentioned that.

Reply:
Thanks!